### PR TITLE
moolticute: 0.30.8 -> 0.42.32-testing

### PR DESCRIPTION
**Pull request by jonringer**
###### Motivation for this change
closes #69670

the -testing variations appear to contain bug fixes according to their releases https://github.com/mooltipass/moolticute/releases

so i grabbed the latest

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/moolticute/default.nix [hunk 1]
```diff
@@ -1,36 +1,44 @@
...
+, libusb1
+, pkg-config
+, qmake
+, qtbase
+, qttools
+, qtwebsockets
 }:
 
-stdenv.mkDerivation rec {
+mkDerivation rec {
```
Comment (danbst) [inline pkgs/applications/misc/moolticute/default.nix]: have I missed something, is this new official mkDerivation style?

Comment (jonringer) [reply pkgs/applications/misc/moolticute/default.nix]: for qt packages, yes. This mkDerivation is actually different than stdenv.mkDerivation.

Comment (danbst) [reply pkgs/applications/misc/moolticute/default.nix]: right! thanks. I wish it was called `mkQtDerivation` https://github.com/NixOS/nixpkgs/issues/65399#issuecomment-526649894

Comment (jonringer) [reply pkgs/applications/misc/moolticute/default.nix]: as do i, it causes a lot of confusion. Or `buildQtApplication`

**Pull request merged**